sdk/azidentity/interactive_browser_credential_test.go

```diff
@@ -5,15 +5,16 @@ package azidentity
 
 import (
 	"context"
-	"errors"
-	"net/http"
+	"os"
 	"testing"
+	"time"
 
 	"github.com/Azure/azure-sdk-for-go/sdk/azcore/policy"
-	"github.com/Azure/azure-sdk-for-go/sdk/internal/mock"
-	"golang.org/x/net/http2"
+	"github.com/AzureAD/microsoft-authentication-library-for-go/apps/public"
 )
 
+var runManualBrowserTests = os.Getenv("AZIDENTITY_RUN_MANUAL_BROWSER_TESTS") != ""
```

Comment:
If you're checking to make sure the value is set you could use 
`var _, runManualBrowserTests = os.LookupEnv("...")`